api/grpc/types/api.proto

```diff
@@ -3,137 +3,138 @@ syntax = "proto3";
 package types;
 
 service API {
-	rpc CreateContainer(CreateContainerRequest) returns (CreateContainerResponse) {}
-	rpc UpdateContainer(UpdateContainerRequest) returns (UpdateContainerResponse) {}
-	rpc Signal(SignalRequest) returns (SignalResponse) {}
-	rpc AddProcess(AddProcessRequest) returns (AddProcessResponse) {}
-	rpc CreateCheckpoint(CreateCheckpointRequest) returns (CreateCheckpointResponse) {}
-	rpc DeleteCheckpoint(DeleteCheckpointRequest) returns (DeleteCheckpointResponse) {}
-	rpc ListCheckpoint(ListCheckpointRequest) returns (ListCheckpointResponse) {}
-	rpc State(StateRequest) returns (StateResponse) {}
-	rpc Events(EventsRequest) returns (stream Event) {}
-	rpc GetStats(StatsRequest) returns (stream Stats) {}
+  rpc CreateContainer(CreateContainerRequest) returns (CreateContainerResponse) {}
+  rpc UpdateContainer(UpdateContainerRequest) returns (UpdateContainerResponse) {}
+  rpc Signal(SignalRequest) returns (SignalResponse) {}
+  rpc AddProcess(AddProcessRequest) returns (AddProcessResponse) {}
+  rpc CreateCheckpoint(CreateCheckpointRequest) returns (CreateCheckpointResponse) {}
+  rpc DeleteCheckpoint(DeleteCheckpointRequest) returns (DeleteCheckpointResponse) {}
+  rpc ListCheckpoint(ListCheckpointRequest) returns (ListCheckpointResponse) {}
+  rpc State(StateRequest) returns (StateResponse) {}
+  rpc Events(EventsRequest) returns (stream Event) {}
+  rpc GetStats(StatsRequest) returns (stream Stats) {}
+}
+
+message IO {
+  string stdin = 1; // path to the file where stdin will be read (optional)
+  string stdout = 2; // path to file where stdout will be written (optional)
+  string stderr = 3; // path to file where stderr will be written (optional)
+  string console = 4; // path to the console for a container (optional)
 }
 
 message CreateContainerRequest {
-	string id = 1; // ID of container
-	string bundlePath = 2; // path to OCI bundle
-	string stdin = 3; // path to the file where stdin will be read (optional)
-	string stdout = 4; // path to file where stdout will be written (optional)
-	string stderr = 5; // path to file where stderr will be written (optional)
-    string console = 6; // path to the console for a container (optional)
-	string checkpoint = 7; // checkpoint name if you want to create immediate checkpoint (optional)
+  string id = 1; // ID of container
```

Comment:
This covers some of it: https://developers.google.com/protocol-buffers/docs/style. Spaces are not covered.

Should we follow https://github.com/google/protobuf/blob/master/conformance/conformance.proto as closely as possible? Seems like 2 space indents but they don't have any convention on lining up fields.

I don't think I am going to be able to work outside of gofmt anymore. It is just too cozy.